e2e/app/page/workspace-edit-page.ts

```diff
@@ -245,6 +245,13 @@ export default class WorkspaceEditPage extends AuthenticatedPage {
     }
   }
 
+  /**
```

Comment:
What do these changes have to do with running tests in staging?

Reply:
result of changes in `Select` class.